modin/db_conn.py

```diff
@@ -28,6 +28,8 @@
 
 
 class UnsupportedDatabaseException(Exception):
```

Comment:
@devin-petersohn, I wonder if this place is appropriate for this file? Should we move it in other place?